src/NUnitFramework/framework/Attributes/TestCaseAttribute.cs

```diff
@@ -250,6 +259,24 @@ public string Category
             }
         }
 
+        /// <summary>
+        /// Gets and sets the ignore until date for this test case.
+        /// </summary>
+        public string Until
```

Comment:
Could this be `DateTimeOffset`?

@nunit/framework-team what do you think?

Reply:
I don't think this can be a `DateTimeOffset` because parameters in an attribute constructor must be `const`. Same as in the `IgnoreAttribute`.

Reply:
Oh, of course... thanks!

Reply:
Oh sorry, quite right!